components/Chat/Message/message.scss

```diff
@@ -22,20 +22,6 @@
   overflow: hidden;
 }
 
-.highlight {
```

Comment:
Why get rid of this? Is it replaced elsewhere?

Reply:
See: fffa60e2032e00d385af0816ce9e29276143b9dc 
```
chore: revert "feat(scrollback): add infinite message scrollback"

This reverts commit 862c1b0, reversing
changes made to 0bf5c20.
```
This commit made things too buggy, crashed a lot, and was hard to develop with.